contracts/zero-ex/contracts/src/features/PancakeSwapFeature.sol

```diff
@@ -0,0 +1,401 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Mostly minor changes here from `UniswapFeature`.
- Remove legacy allowance target logic.
- Remove greedy tokens logic.
- Fortunately the above two were appended logic and don't seem to impact the rest of the assembly by their removal.
- Update names, init hashes, addresses for the new AMMs/chain.

Reply:
Here's a diff of the two files: https://gist.github.com/dorothy-zbornak/5a9d4e883ab1bac1f216354069781163